src/components/layout/inputs/CheckboxInput/CheckboxInput.tsx

```diff
@@ -2,23 +2,35 @@
 * @debt directory "Gaël: this file should be migrated within the new directory structure"
 */
 
-import PropTypes from 'prop-types'
 import React from 'react'
 
+interface IProps {
+  SvgElement?: React.ComponentType | null,
```

Comment:
tu n'as pas besoin du `null` car la prop optionnelle  => ça équivaut à `React.ComponentType | undefined`
```suggestion
  SvgElement?: React.ComponentType,
```

Reply:
Ok on à deux manière de faire :

```
// le type 
propName?: propType

// props par defaut
propName = undefined
```
ou bien 
```
// le type 
propName?: propType | null

// props par defaut
propName = null
```

J'aime bien le `null` explicite.